lms/djangoapps/discussion_api/views.py

```diff
@@ -76,41 +75,30 @@ class CourseTopicsView(DeveloperErrorViewMixin, APIView):
     **Example Requests**:
```

Comment:
@jcdyer Is not **Example Requests** being followed through edx? The `discussion_api` is following it and I see same convention in `course_block_api` and `mobile_api` too.